bundles/org.openhab.core.config.discovery.addon.process/src/main/java/org/openhab/core/config/discovery/addon/process/ProcessAddonFinder.java

```diff
@@ -84,28 +101,30 @@ public Set<AddonInfo> getSuggestedAddons() {
             for (AddonDiscoveryMethod method : candidate.getDiscoveryMethods().stream()
                     .filter(method -> SERVICE_TYPE.equals(method.getServiceType())).toList()) {
 
-                List<AddonMatchProperty> matchProperties = method.getMatchProperties();
-                List<AddonMatchProperty> commands = matchProperties.stream()
-                        .filter(amp -> COMMAND.equals(amp.getName())).toList();
-
-                if (matchProperties.size() != commands.size()) {
-                    logger.warn("Add-on '{}' addon.xml file contains unsupported 'match-property'", candidate.getUID());
-                }
+                Map<String, Pattern> matchProperties = method.getMatchProperties().stream()
+                        .collect(Collectors.toMap(AddonMatchProperty::getName, AddonMatchProperty::getPattern));
 
-                if (commands.isEmpty()) {
-                    logger.warn("Add-on '{}' addon.xml file does not specify match property \"{}\"", candidate.getUID(),
-                            COMMAND);
+                if (matchProperties.isEmpty()) {
+                    logger.warn("Add-on '{}' addon.xml file contains no 'match-property'", candidate.getUID());
```

Comment:
Maybe remove explicit reference to addon.xml (formulate something like Add-on info ... contains no 'match-property').
The reason is we may have other sources of add-on info in the future when we would implement this for the marketplace or 3rd-party providers. These may not be relying on the addon.xml file for this info.